views/driver/driverui.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <link rel="shortcut icon" href="../img/Graphicloads-Transport-Bus-4.ico" type="image/x-icon">
+    <link rel="icon" href="../img/Graphicloads-Transport-Bus-4.ico" type="image/x-icon">
+
+    <link href="https://cdn.metroui.org.ua/v4/css/metro-all.min.css" rel="stylesheet">
+    <link href="../css/mainui.css" rel="stylesheet">
+    <title>Driver UI</title>
+</head>
+
+<body>
+    <div class="container">
+        <aside class="sidebar z-2 csidebar" data-role="sidebar" data-toggle="#sidebar-toggle" id="sb1">
+            <div class="sidebar-header csidebar-header" data-image="../img/bggravatar.png" style="background-image:url("
+                ../img/bggravatar");"> <a href="/" class="fg-white sub-action" onclick="Metro.sidebar.close('#sb1'); return false;"><span
+                    class="mif-arrow-left mif-2x"></span></a>
+                <div class="avatar">
+                    <img data-role="gravatar" src="../img/user.png" data-role-gravatar="true">
+                </div>
+                <span class="title csidebar-title fg-white">Name</span>
+                <span class="subtitle csidebar-subtitle fg-white">Driver-ID</span>
+            </div>
+            <ul class="sidebar-menu">
+                <li><a><span class="mif-home fg-red icon"></span>Home</a></li>
+                <li><a><span class="mif-books icon"></span>Guide</a></li>

```

Comment:
Can the unnecessary icons like Guide be removed?

Reply:
it is just for reference, yes it can be replaced with other icons w.r.t their features